target/linux/ath79/mikrotik/base-files/lib/upgrade/platform.sh

```diff
@@ -7,7 +7,7 @@ platform_check_image() {
 	return 0
 }
 
-RAMFS_COPY_BIN='fw_printenv fw_setenv nandwrite'
+RAMFS_COPY_BIN='fw_printenv fw_setenv nandwrite yafut'
 RAMFS_COPY_DATA='/etc/fw_env.config /var/lock/fw_printenv.lock'
```

Comment:
I think you're right.  a66eee63368eb5625b359244bfffc9b9eb921875 added these lines to `target/linux/ath79/mikrotik/base-files/lib/upgrade/platform.sh` when it moved `platform_do_upgrade_mikrotik_nand()` from `target/linux/ath79/nand/base-files/lib/upgrade/platform.sh`, which also contained these lines.  However, they were added to the latter file by 55e6c903ae20bd8e694f4626166766e59602f87c, which added support for a GL.iNet device, not a Mikrotik one.  I will clean this up as you suggested.

Reply:
Cleaned up in commit da6a964e81f6a246791ab160d8c49b733d46f059, thanks.

Reply:
Oops, I have only just now noticed that the log message for da6a964e81f6a246791ab160d8c49b733d46f059 references a malformed commit hash (`66eee6336` should actually be a66eee6336).  I am going to unresolve this conversation so that I remember to fix this in one of the subsequent PR updates.